$OpenBSD: patch-src_c_ffi_d,v 1.1 2020/02/12 19:42:02 jasper Exp $

libffi for x86_64 doesn't define FFI_SYSV anymore for FFI_UNIX64 should used instead
fixed upstream with https://gitlab.com/embeddable-common-lisp/ecl/commit/b2f09b4809441a92d6c11a2b39d5399580e56ae7

Index: src/c/ffi.d
--- src/c/ffi.d.orig
+++ src/c/ffi.d
@@ -132,10 +132,15 @@ static struct {
   {@':stdcall', FFI_STDCALL},
 #elif defined(X86_WIN64)
   {@':win64', FFI_WIN64},
-#elif defined(X86_ANY) || defined(X86) || defined(X86_64)
+#elif defined(X86_ANY) || defined(X86)
+#if defined(X86_64)
+  {@':cdecl', FFI_UNIX64},
+  {@':sysv', FFI_UNIX64},
+  {@':unix64', FFI_UNIX64},
+#else
   {@':cdecl', FFI_SYSV},
   {@':sysv', FFI_SYSV},
-  {@':unix64', FFI_UNIX64},
+#endif
 #endif
 };
 
